parts/meta/kw_uri_map.txt

```diff
@@ -201,8 +201,20 @@ ENDNUM __RefHeading___Toc123125_83452205
 ENDSCALE __RefHeading___Toc68146_2267116897
 ENDSKIP __RefHeading___Toc605472_3199477706
 ENKRVD __RefHeading___Toc69787_621662414
+ENKRVDX __RefHeading___Toc69787_621662414
```

Comment:
Thanks. The changes to parts/meta/kw_uri_map.txt are intended?

Reply:
Yes. These are inserted by the alias functions. See https://github.com/OPM/opm-reference-manual/pull/410/files#diff-45a6fd9dbaf527c01ddb0d629af97c5b4ecf940c546aa9f3e834044d1985c6dbR135. They create aliases to keywords that are included in another keyword. See https://github.com/OPM/opm-reference-manual/pull/411#issuecomment-2465607885 for more information